.bsp/sbt.json

```diff
@@ -1 +1 @@
-{"name":"sbt","version":"1.6.2","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/jdk1.8.0_161.jdk/Contents/Home/jre/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/kapilsanghvi/Library/Application Support/JetBrains/IdeaIC2021.1/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=/Users/kapilsanghvi/Library/Application%20Support/JetBrains/IdeaIC2021.1/plugins/Scala/launcher/sbt-launch.jar"]}
\ No newline at end of file
+{"name":"sbt","version":"1.6.2","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/temurin-11.jdk/Contents/Home/bin/java","-Xms100m","-Xmx100m","-classpath","/nix/store/rd34r9ksqsl3ikvhdsb747d1b9x4y2yq-sbt-1.6.2/share/sbt/bin/sbt-launch.jar","-Dsbt.script=/Users/andrew/.nix-profile/bin/sbt","xsbt.boot.Boot","-bsp"]}
```

Comment:
**.bsp** folder needs to be deleted I think,  we don't need it. I don't remember why I pushed it